turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
+
+# GCSFuse mount for Turbinia scripts & logging and Google Cloud Functions
+MOUNT_POINT = '/mnt/turbinia'
+
+# Turbinia CLI
+TURBINIACTL = '%s/turbinia/turbiniactl' % SRC_DIR
+
+# Virtualenv directory
+TURBINIAENV = '%s/turbinia-env' % HOME_DIR
+
+# Virtualenv activator
+VIRTUALENV_ACTIVATE = '%s/bin/activate' % TURBINIAENV
+
+# GCS bucket that has Turbinia-specific scripts and
+# that can be used to store logs.
+BUCKET = 'turbinia'
```

Comment:
Buckets are a global namespace, so I would probably set this to None by default (we have a 'turbinia' bucket already in use in one of our projects).

Reply:
Removed as BUCKET_NAME will be used instead.